core/src/test/java/org/elasticsearch/indices/cluster/AbstractIndicesClusterStateServiceTestCase.java

```diff
@@ -303,9 +308,10 @@ public Index index() {
     protected class MockIndexShard implements IndicesClusterStateService.Shard {
         private volatile ShardRouting shardRouting;
         private volatile RecoveryState recoveryState;
+        private volatile long term;
 
-        public MockIndexShard(ShardRouting shardRouting) {
-            this.shardRouting = shardRouting;
+        public MockIndexShard(ShardRouting shardRouting, long term) {
+            this.shardRouting = shardRouting; this.term = term;
```

Comment:
I guess it was folded by intellij into a one liner and I just added it in there. Anyway - fixed.